lib/auth/bento-theme/login/login.ftl

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html <#if realm.internationalizationEnabled> lang="${locale.currentLanguageTag}"</#if>>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="${msg("login.description")}">
+    <title>${msg("login.title")}</title>
+    <link rel="stylesheet" href="${url.resourcesPath}/css/styles.css">
+</head>
+<body>
+    <div class="background">
+        <div class="login-container">
+            <#if realm.internationalizationEnabled && locale.supported?size gt 1>
+                <div class="language-switcher">
+                    <select
+                            aria-label="${msg("languages")}"
+                            id="login-select-toggle"
+                            onchange="if (this.value) window.location.href=this.value">
+                        <#list locale.supported?sort_by("label") as l>
+                            <option
+                                    value="${l.url}"
+                                    ${(l.languageTag == locale.currentLanguageTag)?then('selected','')}>
+                                ${l.label}
+                            </option>
+                        </#list>
+                    </select>
+                </div>
+            </#if>
+
+            <img src="${url.resourcesPath}/img/branding.png" alt="${msg("login.logo.alt")}" class="logo" />
+
+            <form id="kc-form-login" onsubmit="login.disabled = true; return true;" action="${url.loginAction}" method="post" aria-labelledby="login-heading">
```

Comment:
I think there can be messages (e.g., login expired) that are missing from being rendered here

Reply:
<img width="624" alt="Screenshot 2025-01-24 at 7 53 34 AM" src="https://github.com/user-attachments/assets/440b7d92-501c-46d9-a61e-4e7a60eee92a" />